src/main/scala/com/github/sbt/sbom/BomExtractor.scala

```diff
@@ -263,3 +286,11 @@ class BomExtractor(settings: BomExtractorParams, report: UpdateReport, rootModul
   }
 
 }
+
+object BomExtractor {
+  private[sbom] def enrichedPurl(group: String, name: String, version: String, qualifier: Map[String, String] = Map[String, String]()): String = {
```

Comment:
I'm not sure what is 'enriched' about it, I could live with just `purl` - but no strong objections to this